cmd/internal/cli/key_newpair.go

```diff
@@ -31,3 +35,9 @@ var KeyNewPairCmd = &cobra.Command{
 	Long:    docs.KeyNewPairLong,
 	Example: docs.KeyNewPairExample,
 }
+
+// doKeyNewpairCmd is the 'singularity key newpair' command
+func doKeyNewpairCmd(keyServerURI, authToken string) error {
```

Comment:
Why have a helper function when its passing parameters to `GenKeyPair()` and returning the result without any additional logic?

Reply:
This function is unnecessary, it only *was* there for consistent with other file. *(now that I'm looking at it, its probably better to remove this function)*

Reply:
Agreed :+1: